### PR TITLE
GUVNOR-3237: Change the home page to the new one

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Comment by paulovmr** [comment]
Part of an ensemble:
* https://github.com/kiegroup/kie-wb-common/pull/993
* https://github.com/kiegroup/drools-wb/pull/558
* https://github.com/kiegroup/jbpm-form-modeler/pull/124
* https://github.com/kiegroup/optaplanner-wb/pull/190
* https://github.com/kiegroup/kie-wb-distributions/pull/560

**Comment by jhrcek** [comment]
@paulovmr Looks like the build failure is related to @porcelli 's [PR|https://github.com/AppFormer/uberfire/pull/779] merged yesterday in uberfire. Can you please update this PR based on that and rerun the build and if it passes also run full downstream build? I'm really curious to see the the new home page in action :-)

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-api/src/main/java/org/kie/workbench/common/screens/home/model/HomeShortcut.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-api/src/main/java/org/kie/workbench/common/screens/home/model/HomeShortcut.java]: 2017

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="home">
+    <div class="kie-page">
+        <div data-field="container" class="kie-page__content kie-content--bg-image kie-blank-slate">
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.html]: Interesting inconsistent format of class names.. just a comment.

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-api/src/main/java/org/kie/workbench/common/screens/home/model/HomeShortcut.java (hunk 1 above)
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-api/src/main/java/org/kie/workbench/common/screens/home/model/HomeShortcut.java]: Fixed

File: kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.html (hunk 2 above)
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.html]: Apparently they follow [this](https://en.bem.info/methodology/key-concepts/) methodology :)

Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-home/kie-wb-common-home-client/src/main/java/org/kie/workbench/common/screens/home/client/widgets/home/HomeView.html]: I live and learn :-)

**Comment by tomasdavidorg** [comment]
Jenkins execute full downstream build

**Comment by porcelli** [comment]
Jenkins execute full downstream build

**Comment by paulovmr** [comment]
jenkins retest this

**Comment by paulovmr** [comment]
jenkins retest this

**Pull request merged**